README.md

```diff
@@ -149,7 +149,7 @@ Note: there are a few tricky cases with the **compact IDs (CID)**:
 ### Dev
 | Provider            | Compact Id (CIB)  | Host URI                                   |
 |---------------------|-------------------|--------------------------------------------|
-| AnVIL (gen3 hosted) | dg.anv0           | staging.theanvil.io                        |
+| AnVIL (gen3 hosted) | dg.anv0           | jade.datarepo-dev.broadinstitute.org       |
```

Comment:
maybe also change this parenthetical to `(TDR hosted)` or remove it

Reply:
ah good call!